packages/schema/package.json

```diff
@@ -20,6 +20,6 @@
   },
   "homepage": "https://github.com/cydran/cydran#readme",
   "devDependencies": {
-    "ajv": "^8.11.0"
```

Comment:
These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

Reply:
fixing